src/components/login/SocialButton.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
Is this file also required?

Reply:
it is necessary because there are places where it can be reused. So I decided to create a single instance and reused it at these different places by changing the props.